aten/src/ATen/native/SoftMax.cpp

```diff
@@ -154,7 +147,14 @@ Tensor log_softmax_cpu(const Tensor& input_, const int64_t dim_, const bool half
       c10::nullopt /* layout */,
       c10::nullopt /* device */,
       c10::nullopt /* pin_memory */,
-      LEGACY_CONTIGUOUS_MEMORY_FORMAT);
+      at::MemoryFormat::Contiguous);
```

Comment:
Found this, https://fb.workplace.com/groups/pytorch.dev/permalink/543001792944972/. Not sure if revisiting LEGACY_CONTIGUOUS_MEMORY_FORMAT is still on the plan, but you may want to keep the format as is. @eellison @Krovatkin, do you guys know any history about this?

Reply:
ah gotcha. i misunderstood that `define` i'll undo it.